packages/core/src/Provider/TimeSettingsProvider/types.ts

```diff
@@ -0,0 +1,14 @@
+import { TimeSettings } from '../../util';
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore
+import { Provider, ProviderError } from '../..';
+
+export interface TimeSettingsProvider extends Provider {
```

Comment:
```suggestion
export interface CardanoNode {
```
+ move to `core/src/Cardano/types/CardanoNode.ts` 
+ Host and rename method to `StateQuery: { eraSummaries: () => Promise<EraSummary[]> }`

Reply:
Done in d2c0156242b9f6ed7591cee0c0acf75ea940e35e